lint-staged.config.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable filenames/match-regex */
+const escapedFileNames = (filenames) =>
+  filenames.map((filename) => `"${filename}"`).join(' ');
+
+module.exports = {
+  './*.{js,jsx,ts,tsx}': (files) => {
+    const filenames = escapedFileNames(files);
+
+    return [
+      `prettier --with-node-modules --ignore-path='./.prettierignore_staged' --write ${filenames}`,
+      `eslint --no-ignore --max-warnings=0 --fix ${filenames}`,
+      `git add ${filenames}`,
+    ];
+  },
+
+  './*.{json,md,mdx,css,html,yml,yaml,scss}': (files) => {
```

Comment:
Catch-all for everything else. Breaking these out separately allows for the Javascript portion to have precedence (and usually the second part will never get run and reduces cycles).